message/aws_sled.py

```diff
@@ -21,3 +21,10 @@ def s3():
             verify=False
         )
     return boto3.resource('s3')
+
```

Comment:
could you make sure these are all 4 spaces?

Reply:
@abarciauskas-bgse ^

Reply:
Addressed in https://github.com/cumulus-nasa/cumulus-sled/pull/2/commits/40439dbf7a30b69fce504c0cc2644f8cefaaf07a